.github/workflows/production-daily.yml

```diff
@@ -90,6 +91,22 @@ jobs:
 
         continue-on-error: true
 
+      - name: Construct JSON for Slack (Sentry)
+        run: |
+          python lib/sentry_slack.py
+        continue-on-error: true
+      - name: Send latest Sentry issues to Slack
+        id: slack-sentry
+        uses: slackapi/slack-github-action@v2.0.0
```

Comment:
`2.1.0` looks to be the [latest](https://github.com/marketplace/actions/slack-send-to-slack)